### DISCUSSION TITLE
PR merged into nvim-lspconfig; Make sure to update nvim-lspconfig and your nvim config accordingly! Directions are in the README

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#8] [Announcements]
that it.

Mason, brew, helix, zed, and VSCode extension are all on the way!

**Comment by adriangalilea** [comment]
@Feel-ix-343 where to keep track for the helix one?

**Reply by Feel-ix-343** [reply]
Hey! Helix should be working. All you need to do is have the binary in your path. You can run the cargo install command from the readme. Note you must have rust and cargo installed

**Reply by adriangalilea** [reply]
Hello, looks incredible,

Have a few doubts,

> When I make a link with `[[foo]]` how do I jump into it? I tried g-r g-f g-i and none of the commands work.

EDIT: doing g-d works.

Also when I work on `~/notes` the suggestions work just fine but when I move into `~` and write I get none, is it because it's stuck indexing or something?

btw, the g-r for tags is incredible, being able to jump around where I placed a tag is very handy.

EDIT2: seems like it's working erratically, once I deleted a few files I tested with and opened new ones it stopped working, do you want me to file an issue?

**Reply by Feel-ix-343** [reply]
1. Hey so you can use all of the features through the nvim language server protocol. Most will set up their own keymappings to the nvim lsp functions. For example: to get the definition of the link under the cursor use a mapping to `vim.lsp.buf.definition` (I think thats the name). It seems like you are using something like Lazyvim, so check the docs for mappings it makes.

2. And for the directory problem, opening an issue would be great. My initial reaction is that the lsp is not reading your home directory as a note directory-- so it may not be a performance issue. Is your notes directory a git directory? Right now moxide only initializes on git directory

3. Make sure you have the lsp file watcher enabled. Check the readme for instructions on this.

Thanks for reporting and please open an issue. These config issues are certainly something to iron out before we get our of prerelease

**Reply by adriangalilea** [reply]
> It seems like you are using something like Lazyvim

Helix, so not sure how those things apply.

> Is your notes directory a git directory? Right now moxide only initializes on git directory

I see I see, my directory wasn't git, thank you for pointing that out, I think that should be very clear on the `README.md` :)

Thank you, yeah, was considering opening an issue but didn't.

**Reply by Feel-ix-343** [reply]
Its a bug right now; It should work for any cwd; I will update the ls soon.

**Reply by adriangalilea** [reply]
Is there an github issue to keep track of?

**Reply by Feel-ix-343** [reply]
Hey!

It is here: #44

**Comment by David-Else** [comment]
Support was merged into Helix:

https://github.com/helix-editor/helix/pull/9758

https://github.com/helix-editor/helix/wiki/Language-Server-Configurations#markdown-oxide
